frontend/simple/js/backend/hapi.js

```diff
@@ -105,12 +109,17 @@ export class HapiNamespace extends TrustedNamespace {
   async register (name: string, value: string) {
     console.log(`registering name:`, name)
     let response = await request.post(`${process.env.API_URL}/name`).send({name, value})
+    store.dispatch('addUser', {username: name, identity: value})
```

Comment:
The store shouldn't be modified here, but elsewhere. See next comment for related reasoning.